### ISSUE TITLE
Feature requests that are state of the art for file managers Nautilus and Nemo for Linux distributions Linux Mint and Ubuntu like in default filemanager Dolphin (for KDE)

### ISSUE THREAD

**Issue by usergithub18** [linuxmint/nemo#3277]
Feature requests that are state of the art for file managers Nautilus and Nemo for Linux distributions Linux Mint and Ubuntu like in default filemanager Dolphin (for KDE)

Please offer/build about two new copy features for your file-managers Nautilus and Nemo, etc. (default filemanager in Linux distributions like Linux Mint and Ubuntu), that are today state of the art while copying files (like filemanager Dolphin in Linux Distribution KDE, and default file-manager in Windows 10)

0-100%-Display of copied files (till now):

-Visible display of how much percents of files copied(%), while copying files with default file-managers of Linux distributions (like default copy-function in Windows 10 while copying files, etc.).  

-Pause-Button (clicking on/off for pause while copying):

A visible clickable Pause-Button is today state of the art (like default file-manager in Windows 10), while copying files, because it should be possible to offer a pause, to do other things on Computer, etc. (e.g. while copying a lot of files for hours, where also a display of percents of files copied is important for your users)

If no advance built in new versions of Nautilus and Nemo

If no advance feature requests are built in new versions of default file manager Nautilus and Nemo for Linux Mint and Ubuntu distributions, you should/could replace default filemanagers (for Linux distributions Linux Mint and Ubuntu) with state of the art filemanager Dolphin (default filemanager for KDE).

Many thanks.

Kind regards

[mailing188@yahoo.com](mailto:mailing188@yahoo.com)

Link Dolphin filemanager:

https://en.wikipedia.org/wiki/Dolphin_(file_manager)

https://opensource.com/article/22/12/linux-file-manager-dolphin

https://github.com/KDE/dolphin

**Comment by Jeremy7701** [comment]
Why not simply 
### load the file manager of your choice?

**Comment by marcus125** [comment]
There is a 0-100% progress bar pop-up when you copy files (see below screenshots), but for me it only appears if I am copying very large files (at least a 1GB or so in total). For Nemo, there's a stop button next to it, but if you click it then it ends copying immediately and you can't resume. There's also a 'play' button, but clicking it doesn't do anything for me. I don't know if this progress bar only appears for large files, or if only after a certain amount of time. I think the latter is what makes sense, as you don't need to see it if it only takes a few seconds to copy.

Regarding pause/resume functionality, I agree, it'd be nice to have and maybe this is what the 'play' button should be for. Though implementing this is way out of my ability!

Also, nautilus's repository is at https://gitlab.gnome.org/antoniof/nautilus and its issues are at https://gitlab.gnome.org/GNOME/nautilus/-/issues.

This is what nemo has:
![copy-progress-bar-in-nemo](https://github.com/linuxmint/nemo/assets/138035097/e1a9a0ca-5b4d-4653-a7cf-65bdfc4cfe8e)

This is what nautilus has:
![copy-progress-bar-for-nautlus](https://github.com/linuxmint/nemo/assets/138035097/86f468e7-2037-4df1-8268-b8497c0b7754)

**Comment by mtwebster** [comment]
If you start a long file operation, you get the progress window there. If you then start additional long operations, they'll show up there as well, but in a queued state:
![image](https://github.com/linuxmint/nemo/assets/262776/1bfcf633-29af-4b61-ab94-e7078ed631b5)

Once the current operation completes the next one will start automatically. If you don't wish to wait for whatever reason, you can click the play button to start the second operation concurrently. This queuing behavior can be disabled in Preferences - in this case all operations will begin immediately.

Pause/resume is something I've wanted to do, it's just not a priority and not really much asked for. Storage speed is increasing which may be part of it - operations don't last all that long to begin with, except with usb media and remote filesystems.

> If no advance feature requests are built in new versions of default file manager Nautilus and Nemo for Linux Mint and Ubuntu distributions, you should/could replace default filemanagers (for Linux distributions Linux Mint and Ubuntu) with state of the art filemanager Dolphin (default filemanager for KDE).

It's nice to have choices, as a user. Nemo is tightly integrated with Cinnamon. We're not trying to be the most popular or most advanced  file manager out there - we want it to be the best for Cinnamon and Mint, and what we think our user base wants. That it's popular elsewhere is a bonus.

**Comment by Jeremy7701** [comment]
I would guess that copy is initiated via a system function call, asynchronously.
Being able to interrupt it - with the possibility of restarting it without any data loss would appear to be impossible.
I can't imagine a scenario in which you would want to do this, anyway.

BTW nemo can copy "waiting" files - if you tell it to start processing them (although if it's to the same output device, its likely to be an inefficient way of doing things).
